language-support/java/bindings-rxjava/src/test/scala/com/daml/ledger/rxjava/components/tests/helpers/DummyLedgerClient.scala

```diff
@@ -128,7 +134,7 @@ class DummyLedgerClient(
 
   override def getLedgerIdentityClient: LedgerIdentityClient = new LedgerIdentityClient {
     override def getLedgerIdentity: Single[String] = Single.just(ledgerId)
-    override def getLedgerIdentity(accessToken: String): Single[String] = Single.just(ledgerId)
+    override def getLedgerIdentity(accessToken: String): Single[String] = ???
```

Comment:
Please add a comment that explains shortly why this can/should be `???`.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/3653/commits/aa780acad3c08acb02bddd6a4e713678fe556cc3